ttnn/ttnn/types.py

```diff
@@ -76,3 +76,14 @@ class ShardStrategy(Enum):
 class ShardOrientation(Enum):
     ROW_MAJOR = 1
     COLUMN_MAJOR = 2
+
+
+class ShardStrategy(Enum):
```

Comment:
Please remove this. We already have these two classes defined in `ttnn`. See `ttnn/ttnn/__init__.py`. e.g. `ttnn.ShardStrategy.HEIGHT` should already work.